services/forms/repo_form.go

```diff
@@ -498,6 +498,7 @@ type UserCreateProjectForm struct {
 type EditProjectBoardForm struct {
 	Title   string `binding:"Required;MaxSize(100)"`
 	Sorting int8
+	Color   string `binding:"MaxSize(7)"`
```

Comment:
can you add validation to ensure this is a valid hex (as this could be an XSS vector)

Reply:
I did a fix for this, if the color isn't a valid hex I assume the user is trying to do something really strange (like hacking) and I throw a HTTP 500 error. Is that ok ?

Reply:
yes